docs/references/api/resource_embedding.rst

```diff
@@ -5,6 +5,10 @@ Resource Embedding
 
 PostgREST allows including related resources in a single API call. This reduces the need for many API requests.
 
+.. note::
+
+  It is **not** possible to perform mutations *on* related resources. When combining mutations and resource embedding, the mutation is performed first and then the related resource is returned in combination with :ref:`prefer_return`.
```

Comment:
After reading the discussion about the breaking change again, I think we should not document this behavior. This is a bug that needs to be fixed by throwing an error accordingly.

Reply:
Can we consider it a bug if we are actively testing for it (and for a while now)?

https://github.com/PostgREST/postgrest/blob/bccf7ac97a8a4d821fe199cab530ec3d37c6c653/test/spec/Feature/Query/DeleteSpec.hs#L73-L78

Not against removing this behavior, but I think it could still be considered a breaking change due to these tests.